frontend/helpers/getConfig.js

```diff
@@ -0,0 +1,23 @@
+import { logger } from '@shopgate/pwa-core/helpers';
+
+let cached;
+
+/**
+ * Gets extension config or empty object on failure.
+ * @returns {Object}
+ */
+function getConfig() {
```

Comment:
if i remember correct, the only reason why szymon introduced this helper was testing. Since we don't have any tests here 🙈 we don't need this file.

Reply:
getConfig has been removed